{{cookiecutter.project_slug}}/README.md

```diff
@@ -4,14 +4,15 @@ The project description is here.
 
 ## Quick Start
 
-Sign up at [neu.ro](https://neu.ro) and setup your local machine according to [instructions](https://neu.ro/docs).
+Sign up at [neu.ro](https://neu.ro) and setup your local machine according to [instructions](https://docs.neu.ro/).
  
 Then run:
 
 ```shell
+pip install -U neuromation
 neuro login
-make setup
-make jupyter
+neuro-flow build {{ cookiecutter.project_slug }}
```

Comment:
I liked the idea of keeping the image name similar to the project slug, but if you feel that it is better to keep it as `myimage`, let's change it also here.
```suggestion
neuro-flow build myimage
```

Reply:
right! thanks